### DISCUSSION TITLE
Grow and shrink  split pane size

### DISCUSSION THREAD

**Discussion by gldtn** [zed-industries/zed#6656] [Feature Requests]
Is there a command to increase or decrease split pane size I can set a shortcut to? I did a quick look at the default keymap and did not see one. If not can you turn this discussion into a suggestion?

Thanks in advance!

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support

**Comment by gldtn** [comment]
Can someone nuke Discord? I want forums back, lol. Thanks @JosephTLyons, will join :smile:
